cluster/src/main/java/com/hartwig/pipeline/reruns/StartingPoint.java

```diff
@@ -32,53 +32,6 @@ public class StartingPoint {
 
     private final StartingPoints startingPoint;
 
-    enum StartingPoints {
-        BEGINNING(Collections.emptyList()),
-        ALIGNMENT_COMPLETE(List.of(Aligner.NAMESPACE, BamMetrics.NAMESPACE, Flagstat.NAMESPACE, SnpGenotype.NAMESPACE)),
-        CRAM_COMPLETE(concat(ALIGNMENT_COMPLETE.namespaces, List.of(CramConversion.NAMESPACE))),
-        SKIP_GRIDSS(List.of(Aligner.NAMESPACE,
-                BamMetrics.NAMESPACE,
-                GermlineCaller.NAMESPACE,
-                Flagstat.NAMESPACE,
-                SnpGenotype.NAMESPACE,
-                Gridss.NAMESPACE,
-                CramConversion.NAMESPACE,
-                LilacBamSlicer.NAMESPACE)),
-        CALLING_COMPLETE(concat(CRAM_COMPLETE.namespaces,
-                List.of(SageConfiguration.SAGE_SOMATIC_NAMESPACE,
-                        GermlineCaller.NAMESPACE,
-                        Gridss.NAMESPACE,
-                        Cobalt.NAMESPACE,
-                        Amber.NAMESPACE,
-                        SageConfiguration.SAGE_GERMLINE_NAMESPACE,
-                        PaveSomatic.NAMESPACE,
-                        PaveGermline.NAMESPACE,
-                        VirusBreakend.NAMESPACE,
-                        LilacBamSlicer.NAMESPACE))),
-        GRIPSS_COMPLETE(concat(CALLING_COMPLETE.namespaces, List.of(GRIPSS_SOMATIC_NAMESPACE, GRIPSS_GERMLINE_NAMESPACE))),
-        PURPLE_COMPLETE(concat(GRIPSS_COMPLETE.namespaces, List.of(Purple.NAMESPACE))),
-
-        RERUN_532(concat(SKIP_GRIDSS.namespaces,
-                List.of(Cobalt.NAMESPACE,
-                        Amber.NAMESPACE,
-                        SageConfiguration.SAGE_GERMLINE_NAMESPACE,
-                        SageConfiguration.SAGE_SOMATIC_NAMESPACE,
-                        VirusBreakend.NAMESPACE))),
-
-        RESEARCH_AFTER_531_DIAGNOSTIC(concat(ALIGNMENT_COMPLETE.namespaces, List.of(GermlineCaller.NAMESPACE))),
-        BWA_COMPLETE(List.of(Aligner.NAMESPACE));
```

Comment:
`BWA_COMPLETE` was temporary, added only for verification of the changes in 5.33 to use new tooling for the BAM metrics and flagstats. Should not be used again.